### DISCUSSION TITLE
Add option to exclude NPCs from dead NPC hiding

### DISCUSSION THREAD

**Discussion by RonnyTommy548** [runelite/runelite#15136] [Ideas]
Some monsters like nightmare, vetion, ice demon and zalcano do not interact well with the new feature of hiding dead NPCs. I love the new feature to hide dead NPCs, and I want to just leave it on always, but this presents a problem at certain encounters. For nightmare in particular, when you kill nightmare it is considered a dead NPC so the "disturb" option to start another fight is hidden. There are some issues related to this (#15074, #12453). I'm writing this as a request so we can have the convenience of leaving the option on to hide dead NPCs everywhere else in the game and not be required to turn the setting on/off.

**Comment by Nightfirecat** [comment]
We fixed Zalcano with the point release today--restarting your client will cause it to work correctly. As for Nightmare and other NPCs (see #15135), we will look to fix their behavior in the coming week or two. We'd very much like for this feature to not be configurable, but to ultimately "just work" instead.
